internal/commands/waypoint/add-ons/create.go

```diff
@@ -58,8 +58,8 @@ $ hcp waypoint add-ons create -n=my-addon -a=my-application -d=my-addon-definiti
 					Required:     true,
 				},
 				{
-					Name:         "application-name",
-					DisplayValue: "NAME",
+					Name:         "app",
+					DisplayValue: "APP-NAME",
```

Comment:
nit: vaultsecrets were advised to keep it at NAME only

Reply:
Got it, fixed in [fbe2710](https://github.com/hashicorp/hcp/pull/77/commits/fbe271054a7e05d97b1168d5e6ecb166d9ccc9f3)